src/common/navigator-utils.ts

```diff
@@ -8,12 +8,26 @@ export class NavigatorUtils {
 
     public getBrowserSpec(): string {
         const userAgent = this.navigatorInfo.userAgent;
-        const versionOffset = userAgent.indexOf('Chrome');
+
+        const edgeVersion = this.getVersion(userAgent, 'Edg');
```

Comment:
Just checking, is it supposed to be spelled like 'Edg'?

Reply:
@lisli, thanks for checking! Yes, the Edge team chose "Edg" to avoid old browser detection that looks for "Edge". More details [here](https://blogs.windows.com/msedgedev/2019/04/08/microsoft-edge-preview-channel-details/) under the "User Agent String" heading.